source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,356 @@
+      :alt: Revert a migration
+   
+   This will revert your migration. You will need to migrate again if you 
+   want to enable Flexible Sync in the future.
+
+.. _realm-sync-migrate-after:
+
+After Backend Migration
+-----------------------
+
+Your migrated backend uses a partition key field in your Realm object models
+to map the Partition-Based Sync client objects to Flexible Sync equivalents on
+the backend. If your object models don't have a partition key field, the
+backend automatically injects one into every object the Partition-Based Sync
+client creates.
+
+These are some other characteristics of the injected field you should be aware
+of:
+
+- The injected field follows this structure: ``<yourPartitionKey> == 
+  <yourPartitionValue>``. Both the field key is derived from your old
+  Partition-Based Sync  **backend**  configuration. The value is derived from
+  your current Partition-Based Sync  **client**  configuration.
+- The injected field does not sync back to the client that created the object.
+  This means the originating client won't be able to read the injected
+  ``partitionKey`` field.
+
+The Partition-Based Sync client automatically creates sync subscriptions for 
+each table where ``partitionKey == partitionValue``. This will continue
+until you :ref:`migrate your client app <realm-sync-migrate-client>` to
+Flexible Sync.
+
+.. _realm-sync-migrate-client:
+
+Migrate Client App to Flexible Sync
+-----------------------------------
+
+After your app's backend has been migrated to Flexible Sync, we recommend
+that you also migrate your client app to Flexible Sync. When you migrate your
+client app to Flexible Sync, the client will stop automatically creating
+sync subscriptions for each Realm object model in your app's data model.
+
+We also recommend removing all sync subscriptions and then creating new subscriptions
+for your data. This is the clearest way to control what data is synced after a
+migration.
+
+You can keep these subscriptions if you don't want to remove and recreate
+subscriptions. See the SDK documentation for details about adding and removing
+subscriptions.
+
+- Flutter SDK
+   :ref:`Manage Flexible Sync Subscriptions - Flutter SDK <flutter-flexible-sync-manage-subscriptions>`
+
+- Java SDK
+   :ref:`Flexible Sync - Java SDK <java-flexible-sync>`
+
+- Kotlin SDK
+   :ref:`Subscribe to Queryable Fields - Kotlin SDK <kotlin-subscriptions>`
+
+- .NET SDK
+   :ref:`Manage Flexible Sync Subscriptions - .NET SDK <dotnet-flexible-sync>`
+
+- Node.js SDK
+   :ref:`Flexible Sync - Node.js SDK <node-flexible-sync>`
+
+- React Native SDK
+   :ref:`Flexible Sync - React Native SDK <react-native-flexible-sync>`
+
+- Swift SDK
+   :ref:`Manage Flexible Sync Subscriptions - Swift SDK <swift-manage-flexible-sync-subscriptions>`
+
+If you don't remove all subscriptions, the automatically-created subscriptions
+will remain, but you won't see them in your client code. This makes it harder
```

Comment:
"...but you won't see them in your client code" was a little confusing to me. I assumed they would be available in the realm.subscriptions set with the autogenerated names along with any other subscriptions. But after re-reading this I think you mean something like "you don't see a code path in your code base that creates these subscriptions so you might forget these exist." (Which is probably also true of the partition key we automagically create.)

I don't have a specific suggestion, but I wonder if it's worth trying to make clearer that these subscriptions are accessible on the client but because they weren't created in the client codebase, it might make maintaining and extending client code confusing.